lib/destack/.gitignore

```diff
@@ -32,3 +32,5 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+lib/destack/.tsbuildinfo
```

Comment:
```suggestion
.tsbuildinfo
```

Reply:
@JcdeA Don't worry about that little thing, will sort it out later.

Thanks for bringing typescript to Destack!